ui/src/present.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2021 The Radicle Upstream Contributors
+//
+// This file is part of radicle-upstream, distributed under the GPLv3
+// with Radicle Linking Exception. For full terms see the included
+// LICENSE file.
+
+function shorten(
+  value: string,
+  length: number = 5,
+  offset: number = 2
+): string {
+  return `${value.slice(0, length + offset)}…${value.slice(-length)}`;
+}
+
+export const string = shorten;
+export const ethTx = ethAddress;
+
+export function urn(value: string): string {
+  return shorten(value.replace("rad:git:", ""), 8, 0);
+}
+
+export function deviceId(value: string): string {
```

Comment:
How about calling this `shortDeviceId` instead of just `deviceId`? I think it’s important to include this to avoid confusing the user and to disambiguate it from other presentations of a device  ID.

The same goes for all the other function names.

Reply:
ede1ef8fc2128e8df4f4e717bfc45bdead41ed68